data/scribo-test/Makefile

```
@@ -0,0 +1,12 @@
+rebuild:
+	cd data ; \
+	rm -rf OCR-D-PRE* ;\
```

should be covered by `--overwrite`
```suggestion
	rm -rf OCR-D-PRE* ;\
```

Also, for the first time, I think you have to transition `scribo-test/mets.xml` via something along these lines:
```sh
ocrd workspace remove-group -fr OCR-D-IMG-BIN-* OCR-D-SEG-PAGE-*
```

That's probably also help with some of the (new?) errors in `make validate`:
```
FileNotFoundError: File path passed as 'url' to download_to_directory does not exist: OCR-D-IMG-BIN-SAUVOLA/OCR-D-SEG-PAGE-SAUVOLA-orig_tiff-BIN_sauvola.png
```

---

Also, could you please add a target to update the bag checksums? (I have never got that part of the workspace bagger to work for me.)